cmd/tellor/argTypes.go

```diff
@@ -63,10 +63,10 @@ func (b *EthereumInt) Set(v string) error {
 	g := new(big.Int)
 	_, ok := g.SetString(v, 10)
 	if !ok {
-		return errors.Errorf("%s is not a valid integer", v)
+		return errors.Errorf("invalid integer: %v", v)
 	}
 	if len(g.Bytes()) > 32 {
-		return errors.Errorf("%s is larger than 256 bits", v)
+		return errors.Errorf("invalid size: %v is larger than 256 bits", v)
```

Comment:
```suggestion
		return errors.Errorf("invalid size larger than 256 bits:%v", v)
```
it is common practise to put arguments at the end.